doc/README.rst

```diff
@@ -81,7 +81,7 @@ have ``sphinx`` and ``ipython`` installed. `numpydoc
 <https://github.com/numpy/numpydoc>`_ is used to parse the docstrings that
 follow the Numpy Docstring Standard (see above), but you don't need to install
 this because a local copy of ``numpydoc`` is included in the pandas source
-code.
+code. To build the Jupyter notebooks included in the documentation, you'll need `nbconvert <https://nbconvert.readthedocs.io/en/latest/>`_ and `nbformat <http://nbformat.readthedocs.io/en/latest/>`_.
```

Comment:
we might also have this same documenation in ``contributing.rst`` so update there as well (as well as some info on what deps you need to build the docs)